src/ui/components/Tour/Tour.tsx

```diff
@@ -82,17 +99,39 @@ const Tour: React.FC = () => {
     </div>
   );
 
-  const timeTravel = (
-    <div className={styles.intro}>
-      <div className={styles.h1}>Time travel 🚀</div>
-      <p>In DevTools, look underneath the video to find the Replay console.</p>
-      <p>
-        <div className={styles.stepCallout}>
-          Hover in the console and click a button to time travel!
-        </div>
-      </p>
-    </div>
-  );
+  const timeTravel =
+    typeOfReplay === "events" ? (
```

Comment:
```js
const typeOfReplay = recording && isTestSuiteReplay(recording) ? "cypress" : "events";
```

nit: This is an odd enum value; "events" is not a type of Replay.

Reply:
Good call. If I'm reading blame correctly, it looks like you added this 4 months ago:

```
  const typeOfReplay = recording && isTestSuiteReplay(recording) ? "cypress" : "events";
```

So I was following that. Happy to change this, do you want me to say != cypress here instead?

Reply:
Since there's an approval I am going to merge this to get it live, but happy to fix this in a follow-up

Reply:
> If I'm reading blame correctly, it looks like you added this 4 months ago

This is no big deal but it got me curious. It was hard to imagine that I wrote that bc the naming choice seemed so strange to me.

The revision from 4 months ago was just a formatting change. The original lines came from PR #8956. 😆

Reply:
Oh! Got it. That PR is the best PR in the history of PRs.

I'll address this in a follow-up.

Reply:
Hey I'm about to adjust this @bvaughn and I'm curious if you have a naming preference here. I often say "manual replay" or "standard replay." Do you have a term you've used in the code that I can align to like "default" or "standard?"

Reply:
That's a good question!

Maybe we could just side-step the name entirely by using a boolean (`isTestReplay`?) 🤪

Reply:
Sounds groovy, thanks!